src/Umbraco.Web.Common/Extensions/FriendlyImageCropperTemplateExtensions.cs

```diff
@@ -0,0 +1,319 @@
+        /// <param name="upScale">
+        /// If the image should be upscaled to requested dimensions
+        /// </param>
+        /// <returns>
+        /// The <see cref="string"/>.
+        /// </returns>
+        public static string GetCropUrl(
+            this string imageUrl,
+            IImageUrlGenerator ImageUrlGenerator,
+            int? width = null,
+            int? height = null,
+            string imageCropperValue = null,
+            string cropAlias = null,
+            int? quality = null,
+            ImageCropMode? imageCropMode = null,
+            ImageCropAnchor? imageCropAnchor = null,
+            bool preferFocalPoint = false,
+            bool useCropDimensions = false,
+            string cacheBusterValue = null,
+            string furtherOptions = null,
+            ImageCropRatioMode? ratioMode = null,
+            bool upScale = true)
+            => imageUrl.GetCropUrl(
+                ImageUrlGenerator,
+                width,
+                height,
+                imageCropperValue,
+                cropAlias,
+                quality,
+                imageCropMode,
+                imageCropAnchor,
+                preferFocalPoint,
+                useCropDimensions,
+                cacheBusterValue,
+                furtherOptions,
+                ratioMode,
+                upScale
+                );
+
+        /// <summary>
+        /// Gets the ImageProcessor URL from the image path.
+        /// </summary>
+        /// <param name="imageUrl">
+        /// The image URL.
+        /// </param>
+        /// <param name="cropDataSet"></param>
+        /// <param name="width">
+        /// The width of the output image.
+        /// </param>
+        /// <param name="height">
+        /// The height of the output image.
+        /// </param>
+        /// <param name="cropAlias">
+        /// The crop alias.
+        /// </param>
+        /// <param name="quality">
+        /// Quality percentage of the output image.
+        /// </param>
+        /// <param name="imageCropMode">
+        /// The image crop mode.
+        /// </param>
+        /// <param name="imageCropAnchor">
+        /// The image crop anchor.
+        /// </param>
+        /// <param name="preferFocalPoint">
+        /// Use focal point to generate an output image using the focal point instead of the predefined crop if there is one
+        /// </param>
+        /// <param name="useCropDimensions">
+        /// Use crop dimensions to have the output image sized according to the predefined crop sizes, this will override the width and height parameters
+        /// </param>
+        /// <param name="cacheBusterValue">
+        /// Add a serialized date of the last edit of the item to ensure client cache refresh when updated
+        /// </param>
+        /// <param name="furtherOptions">
+        /// These are any query string parameters (formatted as query strings) that ImageProcessor supports. For example:
```

Comment:
Guess we shouldn't really be referring to "ImageProcessor" in the intellisense comments.  And not "ImageSharp" either as it could be replaced with something else in any particular solution.  So perhaps replace these references with "the underlying image processing service" or similar?

There's at least a couple references to this in the comments.